regression/goto-diff/java-properties/test.desc

```diff
@@ -5,6 +5,6 @@ old.jar --json-ui --show-properties --cover location
e": "Test.java",\n              "function": "java::Test.<init>:\(\)V",\n              "line": "6"\n            }\n          },\n          \{\n            "class": "coverage",\n            "coveredLines": "4",\n            "description": "block 3",\n            "expression": "false",\n            "name": "java::Test.<init>:\(\)V.coverage.2",\n            "sourceLocation": \{\n              "bytecodeIndex": "3",\n              "file": "Test.java",\n              "function": "java::Test.<init>:\(\)V",\n              "line": "4"\n            }\n          },\n          \{\n            "class": "coverage",\n            "coveredLines": "4",\n            "description": "block 4",\n            "expression": "false",\n            "name": "java::Test.<init>:\(\)V.coverage.3",\n            "sourceLocation": \{\n              "bytecodeIndex": "5",\n              "file": "Test.java",\n              "function": "java::Test.<init>:\(\)V",\n              "line": "4"\n            }\n          },\n          \{\n            "class": "coverage",\n            "coveredLines": "4",\n            "description": "block 5",\n            "expression": "false",\n            "name": "java::Test.<init>:\(\)V.coverage.4",\n            "sourceLocation": \{\n              "bytecodeIndex": "6",\n              "file": "Test.java",\n              "function": "java::Test.<init>:\(\)V",\n              "line": "4"\n            }\n          },\n          \{\n            "class": "coverage",\n            "coveredLines": "7",\n            "description": "block 6",\n            "expression": "false",\n            "name": "java::Test.<init>:\(\)V.coverage.5",\n            "sourceLocation": \{\n              "bytecodeIndex": "7",\n              "file": "Test.java",\n              "function": "java::Test.<init>:\(\)V",\n              "line": "7"\n            }\n          }\n        \],\n        "sourceLocation": \{\n          "file": "Test.java",\n          "function": "java::Test.<init>:\(\)V",\n          "line": "6"\n        }\n      },\n      \{\n        "name": "java::Test.<clinit>:\(\)V",\n        "properties": \[\n          \{\n            "class": "coverage",\n            "coveredLines": "3",\n            "description": "block 1",\n            "expression": "false",\n            "name": "java::Test.<clinit>:\(\)V.coverage.1",\n            "sourceLocation": \{\n              "bytecodeIndex": "1",\n              "file": "Test.java",\n              "function": "java::Test.<clinit>:\(\)V",\n              "line": "3"\n            }\n          }\n        \],\n        "sourceLocation": \{\n          "file": "Test.java",\n          "function": "java::Test.<clinit>:\(\)V",\n          "line": "3"\n        }\n      }\n    \],\n    "newFunctions": \[\n      \{\n        "name": "java::Test.newfun:\(\)V",\n        "properties": \[ \],\n        "sourceLocation": \{\n          "file": "Test.java",\n          "function": "java::Test.newfun:\(\)V",\n          "line": "18"\n        }\n      }\n    \],\n  
```

Comment:
This is certainly not the desired behaviour. Even a method that consists only of END_FUNCTION should get a coverage goal added.

Reply:
Indeed removing labelled `SKIP`s when those labels were transferred to `END_FUNCTION` was wrong. This is now fixed (and those changes have thus been removed).